static/monorail-2.0.yaml

```diff
@@ -4680,6 +4680,39 @@ paths:
       - Write
       - workflowsModify
     x-swagger-router-controller: workflows
+  /notification/alerts:
```

Comment:
@yyscamper This won't work because the alerts are coming back over https.  The monorail-2.0-sb.yaml needs to be removed anyway.  This is part of the 1.1 API removal story in the Stryker backlog.